.pre-commit-config.yaml

```diff
@@ -107,6 +101,22 @@ repos:
         rev: v4.5.0
         hooks:
               - id: check-json
+      - repo: https://github.com/rapidsai/pre-commit-hooks
+        rev: v0.0.1
+        hooks:
+          - id: verify-copyright
+            files: |
+              (?x)
+                  [.](cmake|cpp|cu|cuh|h|hpp|sh|pxd|py|pyx)$|
```

Comment:
should we add in the rust files to this ?
```suggestion
                  [.](cmake|cpp|cu|cuh|h|hpp|sh|pxd|py|pyx|rs)$|
```

Reply:
That was not present in the old file, but I'll go ahead and add it.